src/System.Buffers/tests/System.Buffers.Tests.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Don't run .NET Framework only tests on non Windows. -->
+    <SkipTests Condition="'$(OS)' != 'Windows_NT'">true</SkipTests>
```

Comment:
Do you mind explaining why this was necessary before and why it's not necessary now?

Reply:
Before, this was done in Build.proj which I removed in favor of only maintaining the solution file.